DataCapturing/Synchronization/Synchronizer.swift

```diff
@@ -70,6 +70,10 @@ public class Synchronizer {
     /// A queue synchronizing access to the Alamofire `NetworkReachabilityManager`.
     private let isReachableCheckingQueue = DispatchQueue.global(qos: .background)
 
+    private var isReachableOnEthernetOrWifi = false
+
+    private var isReachable = false
+
```

Comment:
Since the internal attribute is not reliable, we add our own status attributes and change them on each status change.